test-worker.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+    'use strict'
+    importScripts('dist/db.min.js');
+    var dbName = 'tests';
```

Comment:
In the new tests that I've created there's a `guid` function (global) to avoid test name chases. Please use that to get the db name